src/test/ui/suggestions/option-content-move2.stderr

```diff
@@ -0,0 +1,18 @@
+error[E0507]: cannot move out of `var`, a captured variable in an `FnMut` closure
+  --> $DIR/option-content-move2.rs:9:9
+   |
+LL |     let mut var = None;
+   |         ------- captured outer variable
+...
+LL |         move || {
+   |         ^^^^^^^ move out of `var` occurs here
+LL |
+LL |             var = Some(NotCopyable);
```

Comment:
Interestingly, [suggesting `var = Some(NotCopyable).as_ref();`](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=cc9401f83435dd6acfec19a91d83b5b9) or [`var= Some(&NotCopyable);`](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=ec9d6c5ea6d26f4e73bbdf6042f517cd) could be reasonable, but I wouldn't go out of my way to emit either. The `as_ref()` suggestion is meant for people still getting their bearings with the language, and the kind of code that this PR protects against is unlikely to be written by that kind of user.